lib/modules/modules.go

```diff
@@ -331,17 +332,25 @@ func GetModules() Modules {
 
 // ValidateResource performs additional resource checks.
 func ValidateResource(res types.Resource) error {
+	// todo(lxea): DELETE IN 17
```

Comment:
Only the env var part will be deleted in 17, right? The test escape hatch will stay. You may want to update the comment to reflect that, otherwise someone may nuke the entire thing and then wonder why 80 tests started failing.